linux_os/guide/services/usbguard/usbguard_allow_hid/bash/shared.sh

```diff
@@ -1,8 +1,5 @@
 # platform = multi_platform_fedora,Red Hat Enterprise Linux 8,Oracle Linux 8
 
-if ! grep -Eq '^[ \t]*allow[ \t]+with-interface[ \t]+equals[ \t]+\{[ \t]+03:\*:\*[ \t]+\}[ \t]*$' /etc/usbguard/rules.conf ; then
-	echo "allow with-interface equals { 03:*:* }" >> /etc/usbguard/rules.conf
-fi
-if ! grep -Eq '^[ \t]*allow[ \t]+with-interface[ \t]+equals[ \t]+\{[ \t]+03:\*:\*[ \t]+03:\*:\*[ \t]+\}[ \t]*$' /etc/usbguard/rules.conf ; then
-	echo "allow with-interface equals { 03:*:* 03:*:* }" >> /etc/usbguard/rules.conf
+if ! grep -Eq '^[ \t]*allow[ \t]+with-interface[ \t]+all-of[ \t]+\{[ \t]+03:\*:\*[ \t]+\}[ \t]*$' /etc/usbguard/rules.conf ; then
```

Comment:
Shouldn't be `match-all` instead of `all-of`?
```suggestion
if ! grep -Eq '^[ \t]*allow[ \t]+with-interface[ \t]+match-all[ \t]+\{[ \t]+03:\*:\*[ \t]+\}[ \t]*$' /etc/usbguard/rules.conf ; then
```

Reply:
Thank you, I overlooked this and it is fixed.